frontend/src/main/web/src/components/Tooltip/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import { isRequiredForA11y } from 'react-prop-types'
+import { capitalize } from 'lodash'
+import { colors } from '../../constants/styles'
+
+const Tooltip = ({
```

Comment:
Isn't that the whole point of the readme.md and the styleguide? As documentation for user.